model_compression_toolkit/core/pytorch/back2framework/pytorch_model_builder.py

```diff
@@ -256,6 +293,18 @@ def _get_op_func(self,
         """
         return getattr(self, node.name)
 
+    def _get_activation_quantization_fn(self, node):
+        activation_quantization = self.node_to_activation_quantization_holder.get(node.name)
```

Comment:
Is there a reason you store it in a dictionary? Why not just build the layer and use it on the fly? Is it Pytorch-related?